Util/DoctrineHelper.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the DunglasApiBundle package.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Dunglas\ApiBundle\Util;
+
+/**
+ * Class DoctrineHelper
+ *
+ * @author Théo FIDRY <theo.fidry@gmail.com>
+ */
+class DoctrineHelper
+{
+    /**
+     * @var integer
+     */
+    protected static $aliasCount = 0;
```

Comment:
Ok didn't see that before. It looks odd to me. We can consider it as a global variable...